Tasks/NodeToolV0/nodetool.ts

```diff
@@ -47,36 +47,50 @@ interface INodeVersion {
 //      PATH = cacheDir + PATH
 //
 async function getNode(versionSpec: string, checkLatest: boolean) {
+    let installedArch = osArch;
     if (toolLib.isExplicitVersion(versionSpec)) {
         checkLatest = false; // check latest doesn't make sense when explicit version
     }
 
     // check cache
     let toolPath: string;
     if (!checkLatest) {
-        toolPath = toolLib.findLocalTool('node', versionSpec, osArch);
+        toolPath = toolLib.findLocalTool('node', versionSpec, installedArch);
+
+        // In case if it's darwin arm and toolPath is empty trying to find x64 version
+        if (!toolPath && isDarwinArm(osPlat, installedArch)) {
+            toolPath = toolLib.findLocalTool('node', versionSpec, 'x64');
+        }
     }
 
     if (!toolPath) {
         let version: string;
         if (toolLib.isExplicitVersion(versionSpec)) {
             // version to download
             version = versionSpec;
-        }
-        else {
+        } else {
             // query nodejs.org for a matching version
-            version = await queryLatestMatch(versionSpec);
+            version = await queryLatestMatch(versionSpec, installedArch);
+
+            if (!version && isDarwinArm(osPlat, installedArch)) {
+                // nodejs.org does not have an arm64 build for macOS, so we fall back to x64
+                console.log(`Unable to find Node for platform ${osPlat} and architecture ${installedArch}. Trying to install with Rosetta2`);
```

Comment:
I think that better to place this error message in the localization filles.

Reply:
Added localization files